### PR TITLE
Add information about installing Kyma with production profile

### PR DESCRIPTION
**Pull request by colunira**
**Description**

Changes proposed in this pull request:

- Add information about production profile to the installation section

**Related issue(s)**
See also: #7406

### CODE REVIEW

File: docs/kyma/04-05-use-your-own-domain.md [hunk 1]
```diff
@@ -413,6 +415,9 @@ Get the TLS certificate:
    ```bash
    kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/$KYMA_VERSION/installation/resources/tiller.yaml
    ```
+   >**NOTE**: If you want to use Kyma production profile, see the following documents before next step:
+      >* [Istio production profile](/components/service-mesh/#configuration-service-mesh-production-profile)
+      >* [OAuth2 server production profile](/components/security/#configuration-o-auth2-server-profiles)
```
Comment (kazydek) [inline docs/kyma/04-05-use-your-own-domain.md]:
```suggestion
      >* [OAuth2 server production profile](/components/security/#configuration-o-auth2-server-production-profile)
```

File: docs/kyma/04-04-cluster-installation.md [hunk 2]
```diff
@@ -242,6 +247,10 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
    ```bash
    kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/$KYMA_VERSION/installation/resources/tiller.yaml
    ```
+   
+   >**NOTE**: If you want to use Kyma production profile, see the following documents before next step:
+   >* [Istio production profile](/components/service-mesh/#configuration-service-mesh-production-profile)
+   >* [OAuth2 server production profile](/components/security/#configuration-o-auth2-server-profiles)
```
Comment (kazydek) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
   >* [OAuth2 server production profile](/components/security/#configuration-o-auth2-server-production-profile)
```

File: docs/kyma/04-04-cluster-installation.md [hunk 3]
```diff
@@ -179,6 +181,9 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
 
 2. Provision the cluster using the [Kyma CLI](https://github.com/kyma-project/cli).
 
+   >**NOTE**: Kyma offers a production profile. It requires a different machine type. Specify it using the `--type` flag. For more details, see [this](/components/service-mesh/#configuration-service-mesh-production-profile) document.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/kyma/04-04-cluster-installation.md]: ..assuming there is only one & specific, right?

[the remark this answers is not in the document]
Comment (colunira) [reply docs/kyma/04-04-cluster-installation.md]: Yes, that's true :)

**Pull request merged**